en_US/configuration/configuration.md

```diff
@@ -274,9 +273,25 @@ The primitive types are mostly self-describing, so there is usually not a lot to
 
 :::
 
+### Specialized Data Types
+
+Specialized types are essentially primitive types with additional meaning that is reflected in their names, validation rules, and descriptions.
+
+* `duration` (`duration_s` / `duration_ms`)
+
+    A string that represents a time duration, for example: `10s`, `2.5m`, `1h30m`, `2345ms`, `1W2D`. When precision is specified, finer portions of the duration may be ignored: writing `1200ms` for `duration_s` is equivalent to writing `1s`. It doesn't matter if units are in upper or lower case.
+
+* `bytesize`
+
+    A string that represents a number of bytes, for example: `10B`, `640kb`, `4MB`, `1GB`. Units are interpreted as powers of 1024, and the unit part is case-insensitive.
```

Comment:
no longer needed after https://github.com/emqx/emqx/pull/11936/files#diff-a7824deab40445e98a6a9f50b8c0e9607804f8f2383a12b52957736ac1bac3c6R146-R181

Reply:
Hmm, do these changes mean that there will no longer be `Type: duration` in the configuration manual? I'd argue that a kind of user-friendly type-level description for such types would still be warranted. Would be cool if you could click on the `duration` type name in the doc (or hover some ℹ️) and read it. Right now it seems too cumbersome and error-prone to mention what secret is and what `file://` prefix means under each field description instead, as opposed to only once for the `secret` type.

Reply:
Hi, have any conclusions been made about the contents in this section "Specialized Data Types"? Shall I apply the same changes to the Chinese version based on the current English content?